enterprise/premium-skins-and-icon-packs/index.md

```diff
@@ -99,14 +99,12 @@ Below are some recommended combinations of skins and icon packs:
 * [Outside editor]({{site.baseurl}}/enterprise/premium-skins-and-icon-packs/outside-demo/)
 * [Snow editor]({{site.baseurl}}/enterprise/premium-skins-and-icon-packs/snow-demo/)
 
-### Getting started
+{% assign pluginname = 'Premium Skins and Icon' %}
```

Comment:
Wasn't too sure where to add this, but this page has a section on setting up for self-hosted which refers to the "Pro bundle".

Reply:
removed the quoted paragraph